examples/basics/plotting/mpl_plot.py

```diff
@@ -14,6 +14,7 @@
 import numpy as np
 import sys
 
+# import matplotlib.pyplot as plt
```

Comment:
No, I added it to show that you can just replace the import and it will run in vispy instead of MPL. Maybe clarify that a bit?